### DISCUSSION TITLE
O-RAN split with zmq mode

### DISCUSSION THREAD

**Discussion by Zplusless** [srsran/srsRAN_Project#422] [General Help] [answered]
Hi, I'm using srsRAN ZMQ-mode for testing. But after learning the official doc, I'm still not quite clear on how zmq works:

1. In zmq mode, which task, as shown in the diagram below, is completed before the I/Q signal is handed over for transmission via ZeroMQ? It seems that zmq-mode does not use split-7.2?

2. According to the README.MD of srsRAN_Project, "Split-8 fronthaul is supported via UHD for USRP devices". So only the RF part runs on USRP?

Thanks in advance.

![image](https://github.com/srsran/srsRAN_Project/assets/10062702/63d93ae2-7816-40cb-8dd3-64aa2541d915)

**Comment by andrepuschmann** [comment, marked as answer]
Hey - with ZMQ all high-PHY and low-PHY functions (all of the above figure) are carried out in the DU. Then they are sent as baseband IQ to the other side. But samples are of course not transmitted over the air. This only happens with an SDR, like a USRP. But the data generated at the DU is the same for both, i.e. split8.

**Reply by Zplusless** [reply]
So both UHD and ZMQ works in split-8. Thanks!

**Reply by andrepuschmann** [reply]
Yes
